packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -324,6 +324,7 @@ export const raiseHandButtonSelector: RaiseHandButtonSelector;
 export type ReactionButtonSelector = (state: CallClientState, props: CallingBaseSelectorProps) => {
     checked?: boolean;
     disabled?: boolean;
+    allowed?: boolean;
```

Comment:
As this is just used for the composite, remove from the component bindings. The reason for this is that it is *very* app-specific behavior. So if someone if using raw components we don't want to dictate this behavior. As a rule of thumb, anything in the selector should map directly to a prop on the react-components' interface